src/pretty-dom.js

```diff
@@ -31,6 +31,13 @@ function prettyDOM(
   if (dom.documentElement) {
     dom = dom.documentElement
   }
+  if (!('outerHTML' in dom)) {
+    const domType = typeof dom
+    const domTypeName = domType === 'object' ? dom.constructor.name : domType
```

Comment:
This branch is likely not covered. Also: This will throw on `null` because `typeof null === 'object'`.

Reply:
Nice catch. Thank you for explaining.